app/uk/gov/hmrc/cdsreimbursementclaim/services/ClaimTransformerService.scala

```diff
@@ -607,19 +608,24 @@ object DefaultClaimTransformerService {
       case None                 => invalidNel("Could not format display acceptance date")
     }
 
-  def setMrnDetails(completeClaim: CompleteClaim): Validation[List[MrnDetail]] =
-    completeClaim.maybeDisplayDeclaration match {
+  def multipleClaimsAnswer(completeClaim: CompleteClaim): List[(MRN, ClaimedReimbursementsAnswer)] = {
+    val mrns   = completeClaim.movementReferenceNumber :: completeClaim.associatedMRNsAnswer.getOrElse(List())
+    val claims =
+      completeClaim.claimedReimbursementsAnswer :: completeClaim.maybeAssociatedMRNsClaimsAnswer.getOrElse((List()))
+    mrns.zip(claims)
```

Comment:
To be honest this is the reason why I always wanted Complete claim not to render CYA page, because in that case this request object can have full freedom on have to structure fields own way